addons/purchase_mrp/tests/test_purchase_mrp_flow.py

```diff
@@ -223,7 +223,7 @@ def test_kit_component_cost(self):
             self.component_c,
         ]
 
-        self.assertEqual(sum([k.standard_price * k.qty_available for k in components]), 120 * 1260)
+        self.assertAlmostEqual(sum(k.standard_price * k.qty_available for k in components), 120 * 1260, delta=0.5)
```

Comment:
The default delta is already `0.5`. When you add it it looks like you artificially chose the constraint for the assert to work, just rely on the default value in this case.
```suggestion
        self.assertAlmostEqual(sum(k.standard_price * k.qty_available for k in components), 120 * 1260)
```

Reply:
Maybe I don't use the right version of Python but I don't have delta=0.5 by default but places=7
And in that case, the test doesn't work

`line 226, in test_kit_component_cost
    self.assertAlmostEqual(sum(k.standard_price * k.qty_available for k in components), 120 * 1260)
AssertionError: 151199.6 != 151200 within 7 places (0.39999999999417923 difference)`